reward/doc.go

```diff
@@ -14,5 +14,62 @@
 // You should have received a copy of the GNU Lesser General Public License
 // along with the klaytn library. If not, see <http://www.gnu.org/licenses/>.
 
-// Package reward implements the Klaytn reward system.
+/*
+Reward Package
+Package Reward implements the Klaytn Reward System.
+Klaytn Reward System manages stakingInfo and distributes block reward.
```

Comment:
Klaytn Reward System manages --> The Klaytn Reward System manages

Reply:
reflected